### PR TITLE
Add services versions with network policies

### PR DESCRIPTION
**Pull request by corest**
Towards https://github.com/giantswarm/giantswarm/issues/4033

### CODE REVIEW

File: service/controller/aws/v17/version_bundle.go [hunk 1]
```diff
@@ -27,6 +27,36 @@ func VersionBundle() versionbundle.Bundle {
 				Description: "Fix systemd collector D-Bus connection. https://github.com/giantswarm/kubernetes-node-exporter/pull/44",
 				Kind:        versionbundle.KindFixed,
 			},
+			{
+				Components:  "cluster-autoscaler",
```
Comment (rossf7) [inline service/controller/aws/v17/version_bundle.go]:
```suggestion
				Component:  "cluster-autoscaler",
```

**Comment by MarcelMue** [comment]
Is there anything blocking this PR? I would like to get this merged, then upgrade cluster-autoscaler and test the release :)

**Comment by corest** [comment]
Can't merge this as currently, tiller installation is broken onprem. Not because of networkpolicies, but that will just make situation worth

**Comment by corest** [comment]
Hunted down the issue here https://github.com/giantswarm/k8scloudconfig/pull/554

**Pull request merged**